package/kernelctl/package

```diff
@@ -27,7 +27,7 @@ package() {
 }
 
 configure() {
-    if [[ "$(kernelctl list | tail -n +2 | awk '{print $2}' | grep "vanilla-$(< /etc/version)")" == "" ]]; then
+    if [[ ! -e /opt/usr/share/kernelctl/vanilla-"$(< /etc/version)".tar.bz2 ]]; then
```

Comment:
I personally would prefer to use kernelctl so that the package doesn't have to change if you start using multiple archive versions instead of just `.tar.bz2`

Reply:
OK